Update ADHD dataset in examples to MAIN (#1953)
nilearn/nilearn: doc/whats_new.rst

0.5.2
=====

NEW
---

- New brain development fMRI dataset fetcher
  :func:`nilearn.datasets.fetch_development_fmri` can be used to download

>   naturalistic stimuli movie-watching data. A light-weight dataset implemented

Reviewer: ```suggestion
  movie-watching data in children and adults. A light-weight dataset implemented
```
